src/cpu/x64/jit_gemm_inner_product_utils.cpp

```diff
@@ -334,7 +333,7 @@ jit_pp_kernel_t<isa>::jit_pp_kernel_t(size_t OC, size_t MB, dim_t dst_mb_stride,
                 helper_vmm_idx, eltwise_reserved_gpr_, r14, r15, preserve_gpr,
                 preserve_vmm, PARAM_OFF(post_ops_binary_rhs_arg_vec),
                 PARAM_OFF(dst_orig), dst_md_wrapper, tail_size, opmask_binary,
-                reg_tmp, use_exact_tail_scalar_bcast, prelu_helper_vmm_idx};
+                reg_tmp, use_exact_tail_scalar_bcast};
         static const bcast_set_t enabled_bcast_strategy
                 = {broadcasting_strategy_t::scalar,
                         broadcasting_strategy_t::per_oc,
```

Comment:
So this file would be the same with onednn upstream?

Reply:
The intension was to remove all the custom binary prelu related code, whether there is some extra OV specific code is still unknown.